Lib9c/Model/WorldInformation.cs

```diff
@@ -240,24 +240,16 @@ public bool TryAddWorld(WorldSheet.Row worldRow, out World world)
             return true;
         }
 
-        public bool TryUpdateWorld(WorldSheet.Row worldRow, out World world)
+        public void UpdateWorld(WorldSheet.Row worldRow)
         {
-            if (worldRow is null ||
```

Comment:
Why did you remove the `null` check?

Reply:
`UpdateWorld` called only in `HackAndSlash`, and null check before call `UpdateWorld`.